app/src/main/res/layout/fragment_note.xml

```diff
@@ -31,7 +32,7 @@
                 android:layout_width="wrap_content"
                 android:layout_height="wrap_content"
                 android:textStyle="italic"
-                android:text="123"
+                android:text=""
```

Comment:
lets make this: tools:text="123" 
that way it will be shown in android studio, but wont be rendered in the app

Reply:
What's the purpose of setting this attribute? It is overwritten anyway. Am I missing something?

Reply:
I added tools:text, in case you missed it because I edited the file meanwhile.

Reply:
the purpose is just to see that view in the editor, in android studio

Reply:
Ah, I get it now.